link.go

```diff
@@ -28,6 +28,14 @@ type LinkAttrs struct {
 	Namespace    interface{} // nil | NsPid | NsFd
 }
 
+// NewLinkAttrs returns LinkAttrs structure filled with default values
+func NewLinkAttrs() LinkAttrs {
+	return LinkAttrs{
+		TxQLen: -1,
+		MTU:    -1,
+	}
+}
+
```

Comment:
Looks like TxQLen == 0 has important special meaning in some drivers. Actually, same for MTU (basically says default to lower dev MTU). I guess we need `NewLinkAttrs`. @vishvananda thoughts?